collectors/charts.d.plugin/apcupsd/apcupsd.chart.sh

```diff
@@ -41,13 +41,13 @@ apcupsd_check() {
 
 	local host working=0 failed=0
 	for host in "${!apcupsd_sources[@]}"; do
-		run apcupsd_get "${apcupsd_sources[${host}]}" >/dev/null
+		apcupsd_get "${apcupsd_sources[${host}]}" >/dev/null
 		# shellcheck disable=2181
 		if [ $? -ne 0 ]; then
 			error "cannot get information for apcupsd server ${host} on ${apcupsd_sources[${host}]}."
 			failed=$((failed + 1))
 		else
-			apcupsd_status = "$(apcupsd_get ${apcupsd_sources[${host}]} | awk '/^STATUS.*/{ print $3 }')"
+			apcupsd_status=$(apcupsd_get ${apcupsd_sources[${host}]} | awk '/^STATUS.*/{ print $3 }')
```

Comment:
Why remove the quotes here?
I would instead add quotes to $apcupsd_status on the if-else

Reply:
I thought that setting a variable `apcupsd_status` to the output of a command required that there be no spaces between the equals sign. I've also not seen it done with quotes around the $(commands).

Reply:
when assigning the variables we follow the code style like so. We add the quotes.
In the if-else on the other hand, is mandatory to avoid breaking the script in the peculiar case the variable is empty.

Reply:
Thanks for the explanation @paulkatsoulakis. So the proper fix is going to be:
```
apcupsd_status="$(apcupsd_get ${apcupsd_sources[${host}]} | awk '/^STATUS.*/{ print $3 }')"
if [ "${apcupsd_status}" != "ONLINE" ]  && [ "${apcupsd_status}" != "ONBATT" ]; then
```
Adding the quotes back in to line 50 and removing the spaces around the equals sign.
Adding quotes on the test variables on line 51
If that is OK I will make a new pull request with the changes.

Reply:
no need to make new pull request, you can just update your branch and push